packages/app-mobile/components/screens/ConfigScreen/SectionSelector.tsx

```diff
@@ -46,35 +47,38 @@ const SectionSelector: FunctionComponent<Props> = props => {
 		) : null;
 
 		return (
-			<Pressable
+			<TouchableRipple
 				key={section.name}
 				role='tab'
 				aria-selected={selected}
 				onPress={() => props.openSection(section.name)}
-				style={selected ? styles.selectedSidebarButton : styles.sidebarButton}
 			>
-				<Icon
-					name={icon}
-					accessibilityLabel={null}
-					style={styles.sidebarIcon}
-				/>
-				<View style={{ display: 'flex', flexDirection: 'column', flex: 1 }}>
-					<Text
-						numberOfLines={1}
-						style={titleStyle}
-					>
-						{label}
-					</Text>
-					<Text
-						style={styles.sidebarButtonDescriptionText}
-						numberOfLines={1}
-						ellipsizeMode='tail'
-					>
-						{shortDescription ?? ''}
-					</Text>
+				<View
```

Comment:
A `TouchableRipple` can have only one direct child. As such, its contents are now wrapped in a `View`.